src/configuration/locales/fr/translations.json

```diff
@@ -1,3 +1,33 @@
 {
-  "Take the survey!": "Réponder au questionnaire!"
+  "...Loading": "...Chargement",
+  "Beekeeper": "Apiculteur",
+  "Beginner": "Débutant",
+  "Billed: {{value}}": "Payant: {{value}}",
+  "Description {{description}}:": "Description: {{description}}",
+  "Experience: {{experience}}": "Expérience: {{experience}}",
+  "Expert": "Expert",
+  "FreeSpace": "Espace disponible",
+  "Garden": "Jardin",
+  "HAPPYCULTEUR © COPYRIGHT 2019. ALL RIGHTS RESERVED.": "Happyculteur © COPYRIGHT 2019. TOUS DROITS RÉSERVÉS.",
+  "Happyculteur partner!": "Partenaire Happyculteur!",
+  "Individual": "Particulier",
+  "Install": "Installer des ruches",
+  "Interest:": "Intérêt:",
```

Comment:
Write it plurals? `Interests`/`Intérêts`

Reply:
Didn't see your comment before mine. Comment for the english locales 👇🏻